testsuite/generate-format/src/compute.rs

```diff
@@ -28,7 +28,7 @@ fn main() {
     if options.record {
         match output_file {
             Some(path) => {
-                let mut f = File::create("testsuite/generate-format/".to_string() + path).unwrap();
+                let mut f = File::create(path).unwrap();
```

Comment:
I'm pretty sure this is important unless you're moving the files or the input path

Reply:
Sorry yeah! was looking to not merge this pr ever